OutOfSchool/OutOfSchool.BusinessLogic/Models/ContactInfo/ContactsAddressDto.cs

```diff
@@ -1,19 +1,27 @@
-using System.ComponentModel.DataAnnotations;
-using System.Diagnostics.CodeAnalysis;
 using Elastic.Clients.Elasticsearch;
+using OutOfSchool.BusinessLogic.Enums;
 using OutOfSchool.BusinessLogic.Models.Codeficator;
+using OutOfSchool.BusinessLogic.Validators;
 using OutOfSchool.Services.Models.ContactInfo;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
 
 namespace OutOfSchool.BusinessLogic.Models.ContactInfo;
 
 public sealed class ContactsAddressDto : IContentComparable<ContactsAddress>, IEquatable<ContactsAddressDto>
 {
     [Required(ErrorMessage = "Street is required")]
+    [MinLength(1)]
     [MaxLength(60)]
+    [RegularExpression(@"^[0-9�-��-�'.\-\(\)]+$", ErrorMessage = "Field must contain only numbers, Cyrillic letters, and the following symbols: ' . - ( )")]
```

Comment:
As you have written the code on windows - github can't process the unicode encoding and probably our test/prod env also will be affected. Can you try using [Unicode block in hex](https://www.unicode.org/charts/PDF/U0400.pdf) or via the unicode character properties `\p{IsCyrillic}` for .Net regex